src/python/WMComponent/TaskArchiver/TaskArchiverPoller.py

```diff
@@ -233,7 +233,7 @@ def completeTasks(self, finishedwfs):
                     # Tier-0 case, the agent has to mark it completed
                     if not self.useReqMgrForCompletionCheck:
                         self.requestLocalCouchDB.updateRequestStatus(workflow, "completed")
-                        logging.info("status updated to completed %s", workflow)
+                        logging.debug("status updated to completed %s", workflow)
```

Comment:
This transition happens only once per workflow, so I think it's always a good idea to keep it info.

Reply:
production agent doesn't actually reach this code, since it uses reqmgr for the status transition. Unless I missunderstood you?

Reply:
Yes you are right. It is only for Tier0